hnn_core/parallel_backends.py

```diff
@@ -12,10 +12,36 @@
 from warnings import warn
 from subprocess import Popen
 import selectors
+import binascii
 
 _BACKEND = None
 
 
+def _clone_and_simulate(net, trial_idx, prng_seedcore_initial):
+    """Run a simulation including building the network
+
+    This is used by both backends. MPIBackend calls this in mpi_child.py, once
+    for each trial (blocking), and JoblibBackend calls this for each trial
+    (non-blocking)
+    """
+
+    # avoid relative lookups after being forked (Joblib)
+    from hnn_core.network_builder import NetworkBuilder
+    from hnn_core.network_builder import _simulate_single_trial
+
+    # XXX this should be built into NetworkBuilder
+    # update prng_seedcore params to provide jitter between trials
+    for param_key in net.params['prng_*'].keys():
+        net.params[param_key] += trial_idx
+
+    neuron_net = NetworkBuilder(net)
+    dpl = _simulate_single_trial(neuron_net, trial_idx)
+
+    spikedata = neuron_net.get_data_from_neuron()
+
+    return dpl, spikedata
+
+
```

Comment:
@jasmainak @rythorpe Note that this code is shared between MPIBackend and JoblibBackend. Should allow for easier modification of how simulations get seeded in the future

Reply:
This is great @blakecaldwell, thanks!

Where is the `prng_seedcore_initial` argument used in this function? I'm a little surprised that all the tests pass without `_clone_and_simulate()` referencing the initial seedcore params when setting new seedcore params for each trial.

Reply:
Oops! Thanks for the suggestion below.